### PR TITLE
Implemented italian translation

### PR DESCRIPTION
**Pull request by Pyr0x1**
Added italian translation for strings that were still in TODO. I had some troubles with the **sub.php** file, which still has some TODOs, but almost everything is translated now. This PR refers issue #4658. If this PR will be merged can it be accepted with the `hacktoberfest-accepted` label? Thank you.

### CODE REVIEW

File: app/i18n/it/sub.php [hunk 1]
```diff
@@ -12,78 +12,78 @@
...
 				'feed_title' => array(
-					'_' => 'feed title',	// TODO
-					'help' => 'Example: <code>//title</code> or a static string: <code>"My custom feed"</code>',	// TODO
+					'_' => 'titolo del feed',
+					'help' => 'Esempio: <code>//titolo</code> o una stringa statica: <code>"Il mio feed personalizzato"</code>',
 				),
-				'help' => '<dfn><a href="https://www.w3.org/TR/xpath-10/" target="_blank">XPath 1.0</a></dfn> is a standard query language for advanced users, and which FreshRSS supports to enable Web scraping.',	// TODO
+				'help' => '<dfn><a href="https://www.w3.org/TR/xpath-10/" target="_blank">XPath 1.0</a></dfn> è un linguaggio di ricerca standard per utenti avanzati supportato da FreshRSS per abilitare il Web scraping.',
 				'item' => array(
 					'_' => 'finding news <strong>items</strong><br /><small>(most important)</small>',	// TODO
```
Comment (Alkarex) [inline app/i18n/it/sub.php]: There are a few (20) missing  `// TODO` such as this line, if you feel like completing this file

File: app/i18n/it/sub.php [hunk 2]
```diff
@@ -178,32 +178,32 @@
...
-		'add_category' => 'Add a category',	// TODO
-		'add_dynamic_opml' => 'Add dynamic OPML',	// TODO
-		'add_feed' => 'Add a feed',	// TODO
-		'add_label' => 'Add a label',	// TODO
-		'delete_label' => 'Delete a label',	// TODO
+		'add' => 'Aggiungi un feed o una categoria',
+		'add_category' => 'Aggiungi una categoria',
+		'add_dynamic_opml' => 'Aggiungi OPML dinamico',
+		'add_feed' => 'Aggiungi un feed',
+		'add_label' => 'Aggiungi un\'etichetta',
```
Comment (Alkarex) [inline app/i18n/it/sub.php]:
```suggestion
		'add_label' => 'Aggiungi un’etichetta',
```

Favour the typographic apostrophe `’` instead, which also has the advantage of not having to be prefixed by a back-slash, in addition of being nicer

**Comment by Alkarex** [comment]
Thanks and welcome! I have left a few comments higher up

**Review by arnab078 — APPROVED** [review]

File: app/i18n/it/sub.php (hunk 1 above)
Comment (Pyr0x1) [reply app/i18n/it/sub.php]: These are the ones I had some problems with, could you please tell me where to find these strings in the demo so I can have a bit of context on how they are used? Thanks

Comment (Alkarex) [reply app/i18n/it/sub.php]: See for instance:

image

**Comment by Alkarex** [comment]
@Pyr0x1 I have made a few more changes. Could you please double-check? https://github.com/FreshRSS/FreshRSS/pull/4694/commits/fd59b39765fa9d488d9b4483e6af865e0ffd61f9

**Comment by Pyr0x1** [comment]
> @Pyr0x1 I have made a few more changes. Could you please double-check? [fd59b39](https://github.com/FreshRSS/FreshRSS/commit/fd59b39765fa9d488d9b4483e6af865e0ffd61f9)

It's ok, I didn't translate **username** and **default**, because you can often find them untranslated, but your changes are good 👌🏼

**Pull request merged**